### PR TITLE
50years/confetti

### PR DESCRIPTION
**Pull request by narendasan**
Changes for the easter egg and logo fixes

### CODE REVIEW

**Comment by narendasan** [comment]
outstanding bug, canvas is window height and not document height

File: app/views/_partials/acmsh/acmsh.ejs [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+                        var next = this.history.length - this.history_pos;
+                        this.history.push("")
+                        this.input = this.history[next];
+                        this.history.pop();
+                    }
+                    break;
+                
+                case "ArrowLeft":
+                    if (this.cursor_pos > 0) {
+                        this.cursor_pos -= 1
```
Comment (sameetandpotatoes) [inline app/views/_partials/acmsh/acmsh.ejs]: I'm gonna make an issue about this, but we should probably have linting for frontend.

Can you add semicolons to the end of the lines though in the meantime?

File: app/views/_partials/acmsh/acmsh.ejs [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+                    break;
+                case "Control":
+                    break;
+                case "Alt":
+                    break;
+                case "Meta":
+                    break;
+                case "Shift":
+                    break;
+                case "Shift":
```
Comment (sameetandpotatoes) [inline app/views/_partials/acmsh/acmsh.ejs]: Don't think you meant to have multiple `case "Shift"`?

File: app/views/_partials/confetti/ConfettiContext.ejs [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        this.update();
+    }
+    
+    this.stop = function() {
+        cAF(this.interval);
+        this.canvas.style.display = 'none';
+    }
+
+    this.update = function() {
+        console.log("HWHW");
```
Comment (sameetandpotatoes) [inline app/views/_partials/confetti/ConfettiContext.ejs]: Probably can remove this now

File: app/views/_partials/confetti/Constants.ejs [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+ribbonPaperThick = 8.0,
+confettiPaperCount = 95,
+balloonCount = 10,
+radius = 50,
+width_factor = 0.0333;
+height_factor = 0.4;
+tie_width_factor = 0.12;
+tie_height_factor = 0.10;
+tie_curve_factor = 0.13;
+graident_factor = 0.3;
```
Comment (sameetandpotatoes) [inline app/views/_partials/confetti/Constants.ejs]: NP, should be `gradient`

File: public/less/components/footer.less [hunk 5]
```diff
@@ -75,3 +75,14 @@ img.logo#illinois {
     margin-top: 7px;
     float: left;
 }
+
+#confetti{
```
Comment (sameetandpotatoes) [inline public/less/components/footer.less]: NP, should be `#confetti {`

File: app/views/_partials/confetti/Constants.ejs (hunk 4 above)
Comment (narendasan) [reply app/views/_partials/confetti/Constants.ejs]: @sameetandpotatoes what can you clarify this?

Comment (sameetandpotatoes) [reply app/views/_partials/confetti/Constants.ejs]: It's just spelled wrong, not a big deal

**Pull request merged**